### DISCUSSION TITLE
Packaging options for Linux binary

### DISCUSSION THREAD

**Discussion by baqwas** [arduino/lab-micropython-installer#30] [Ideas]
This note is more suited to a Discussion section than Issues.

I understand that the .**deb** (for Linux users) file does the job but would it be better to entertain the concept of **flatpack** (or snap) so that the latest version can be simply downloaded and run (after setting the execution mode).

Some of us prefer to run with the least set of privileges, and offering a .deb file makes us jump through hoops to install .deb files because our working folders are not shared either. Of course, Windows users are lucky (at least in this respect).

Regards.

**Comment by sebromero** [comment]
This is definitely an option. The available makers can be found here: https://www.electronforge.io/config/makers
As far as I can see there should be no issues running the app sandboxed. We're migrating the app to use libusb but I don't think that will cause troubles.

**Comment by baqwas** [comment]
Thanks (for _jogging_ my task list)! Time for me to get back to working on Arduino projects sooner rather later. Been busy with other projects recently.

Regards.
